metadata.yaml

```diff
@@ -9,3 +9,6 @@ releaseSeries:
   - major: 0
     minor: 5
     contract: v1beta1
+  - major: 0
+    minor: 4
+    contract: v1beta1
```

Comment:
@vishesh92 
what's the intention of this change ? to support upgrade from 0.4.x ?